api/include/opentelemetry/trace/global_provider.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <atomic>
+
+#include "opentelemetry/nostd/unique_ptr.h"
+#include "opentelemetry/trace/noop.h"
+#include "opentelemetry/trace/tracer_provider.h"
+
+using opentelemetry::trace::NoopTracer;
+using opentelemetry::trace::TracerProvider;
```

Comment:
Please don't add using statements like this in a header file. It exposes NoopTracer and TracerProvider to anything that includes this header.